RoboFile.php

```diff
@@ -216,7 +216,7 @@ function (Event $event) {
    * @param string $tag
    *   The tag name in the current repository.
    * @param string $branch_name
-   *   The branch name from Pantheon repository. Default: master.
+   *   The branch name from Pantheon repository. Default: master
```

Comment:
Maybe change to `main`?

```suggestion
   *   The branch name from Pantheon repository. Default: main
```

Reply:
Pantheon did not rename the default branches AFAIK, only GitHub.

Reply:
https://discuss.pantheon.io/t/changing-primary-branch-name/1305/18